log/slog/logger.go

```diff
@@ -0,0 +1,47 @@
+// Package slog contains a Logger type that satisfies [cosmossdk.io/log.Logger],
+// backed by a standard library [*log/slog.Logger].
+package slog
+
+import (
+	"log/slog"
+
+	"cosmossdk.io/log"
+)
+
+var _ log.Logger = Logger{}
+
+// Logger satisfies [log.Logger] with logging backed by
+// an instance of [*slog.Logger].
+type Logger struct {
+	log *slog.Logger
+}
+
+// FromSlog returns a Logger backed by an existing slog.Logger instance.
```

Comment:
I would add here that it is the caller responsibility to handle log filtering, and configuration of that logger.

Reply:
Maybe let's just call it `NewCustomLogger` like we did for log, when you pass an already instantiated zerolog

Reply:
Pushed in 8f1f86e927.